zephyr/soc/intel_ace15_mtpm/xtensa/config/tie-asm-cavs25.h

```diff
@@ -0,0 +1,376 @@
+/*
+ * tie-asm.h -- compile-time HAL assembler definitions dependent on CORE & TIE
```

Comment:
tie-asm-cavs25.h seems out-of-place in intel_ace15_mtpm folder?

Reply:
Well... Check content of the zephyr/soc/intel_ace15_mtpm/xtensa/config/tie-asm.h header file.
It looks like it needed I am not sure why and that comment does not explain it anyhow.
The question is why commercial Xtensa compiler (not zephyr sdk) is using cavs_25 tie-asm header but I think it would be best to ask the author.